packages/fxa-content-server/tests/server/routes/validate-email-domain.js

```diff
@@ -57,7 +57,24 @@ registerSuite('routes/validate-email-domain', {
           assert.isTrue(res.json.calledOnceWith({ result: 'MX' }));
         });
       },
-
+      'responds with {result: "skip"} when there is a skip record': () => {
+        const dns = {
+          ...ERROR_CODES,
+          //Promises always resolved so no need to give a Resolver function
+        };
+        const validateEmailDomainRoute = proxyquireWithDns(dns);
+        const req = { query: { domain: 'skip.abc.xyz' } };
+        const res = { json: sinon.stub() };
+        const route = validateEmailDomainRoute();
+        return route
+          .process(req, res)
+          .then(() => {
+            assert.isTrue(res.json.calledOnceWith({ result: 'skip' }));
+          })
+          .catch((err) => {
+            assert.fail(res.json.calledOnceWith(new Error(err.message)));
```

Comment:
@chenba  I was having a thought that we can return error message if the `route` fails due to internal server error so we ca return err.message

Reply:
You don't need a `catch` since a rejection will be a test failure.  But if you want to keep it, pass a message about the failure into `assert.fail`.  The code you added did not throw any errors, so there isn't a particular error to check here.

What sort of "internal server error" do you think we should handle?